script/admin-db-to-json.py

```diff
@@ -289,6 +291,26 @@ def addresses_for_court(self, slug):
         } for row in cur.fetchall()]
         return addresses
 
+    def emergency_message(self):
+        self.logger.info("emergency_message: Exporting emergency_message...")
+        emergency_message = {}
+        cur = self.conn.cursor()
+        cur.execute("SELECT id, show, message FROM emergency_messages")
+        rows = cur.fetchall()
+        if len(rows) == 1:
+            pk, show, message = rows[0]
+            emergency_message = {
+                "show": show,
+                "message": message
+            }
+            message = ("+ %s" % message)
+            self.logger.debug("emergency_message: Added emergency_message, '{}'".format(message))
+        else:
+            self.logger.debug('Error exporting emergency message!!!')
```

Comment:
how about to make this message a bit more clear so we know why the script failed?
maybe something like: 
```
Error exporting emergency message!!! Zero or multiple emergency message records found. Should be only one.
```
feel free to adjust it as you like :)